# This file is part of the RonDB REST API Server
# Copyright (c) 2025 Hopsworks AB
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3.
#
# This program is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program. If not, see <http://www.gnu.org/licenses/>.

Remove install logic from drogon build since we only link to its libraries.

diff --git a/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/CMakeLists.txt b/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/CMakeLists.txt
index e0e35a291ae..3edf4ed92a1 100644
--- a/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/CMakeLists.txt
+++ b/storage/ndb/rest-server2/extra/drogon/drogon-1.9.7/CMakeLists.txt
@@ -844,12 +844,6 @@ install(FILES
     DESTINATION "${INSTALL_DROGON_CMAKE_DIR}"
     COMPONENT dev)
 
-# Install the export set for use with the install-tree
-install(EXPORT DrogonTargets
-    DESTINATION "${INSTALL_DROGON_CMAKE_DIR}"
-    NAMESPACE Drogon::
-    COMPONENT dev)
-
 # Doxygen documentation
 find_package(Doxygen OPTIONAL_COMPONENTS dot dia)
 if(DOXYGEN_FOUND)
